src/index.js

```diff
@@ -10,8 +10,8 @@ export default function image(options = {}) {
 	const filter = createFilter(options.include || includes, options.exclude);
 	let images = [];
 
-	function generateBundle(options, rendered) {
-		const dir = options.dir || dirname(options.dest || options.file);
+	function generateBundle(bunldeOptions, rendered) {
+		const dir = options.dir || dirname(bunldeOptions.dest || bunldeOptions.file);
```

Comment:
This appears to be equivalent?

Reply:
use `bunldeOptions` to prevent `options` params of `createFilter` not be override by `generateBundle`

Reply:
Ah I see, this is potentially a variable shadowing issue. I misread before. 🤔 

You actually want the local `options: OutputOptions` param here. The outer `options` variable is the plugin options.

The naming/shadowing is definitely confusing here. I will change those.

I don't think there's a bug here, though. Could you add a failing test case to demonstrate the issue?

Reply:
if we don't rename the inner `option` to another name,  `options.dir` can not be assign to `const dir` correctly, which make user cannot set output's path.

Reply:
This plugin does not provide a plugin level option `dir`. That is configured via your rollup config, which comes from the `options` param passed to `generateBundle` by rollup.

This plugin currently only writes files to the rollup output directory, no option for a configurable subdirectory is provided. Is that what you are looking for?